torch/nn/attention/flex_attention.py

```diff
@@ -310,16 +294,23 @@ def __str__(self):
 
     def __getitem__(self, index) -> "BlockMask":
         new_values = [x[index] if isinstance(x, Tensor) else x for x in self.as_tuple()]
```

Comment:
is new values used still?
Not sure where this is used since this will regen the Q meta info and is on critical path

Reply:
oops, good catch